tests/conn_ip/Makefile

```diff
@@ -1,4 +1,5 @@
 APPLICATION = conn_ip
+include ../Makefile.tests_common
 
 BOARD ?= native
```

Comment:
Here and below (and probably in other apps that are not included in this PR): This is now redundant, as well as the setting of `QUIET` and `RIOTBASE`. We could do it in a follow-up, but while you are at it you can remove those.